docs/learning/WriteAConnector.md

```diff
@@ -404,27 +358,51 @@ The framework will automatically detect and delete elements and models if the co
 
 ### Getting started
 
-You'll need Node.js version ">=12.22.0 <17.0”. Please refer to [Getting Started](../getting-started/index#getting-started) for more details.
+You'll need Node.js version ">=14.17.0 <17.0”. Please refer to [Getting Started](../getting-started/index#getting-started) for more details.
 
 The node packages you'll need can be installed using
 
 ```Shell
-$npm install  @itwin/core-bentley
-$npm install  @itwin/ecschema-metadata
-$npm install  @itwin/core-geometry
 $npm install  @itwin/core-backend
 $npm install  @itwin/core-common
+$npm install  @itwin/core-bentley
+$npm install  @itwin/node-cli-authorization
 
 $npm install  --save-dev @itwin/build-tools
+$npm install  --save-dev @itwin/core-backend
```

Comment:
so core bentley & backend are both a real + dev dep?

Reply:
can someone review if the deps here make sense for ppl writing their own connector? like the testing, linting, pkgs, etc

Reply:
@aruniverse just updated the npm install to match an actual connector successfully using the new connector-framework.